Manual/HTML/Logical Color Combinations.html

```diff
@@ -0,0 +1,450 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="initial-scale=1">
+    <title>Logical Color Combinations — Keep Talking and Nobody Explodes Module</title>
+    <link rel="stylesheet" type="text/css" href="css/font.css">
+    <link rel="stylesheet" type="text/css" href="Css/normalize.css">
```

Comment:
```suggestion
    <link rel="stylesheet" type="text/css" href="css/normalize.css">
```
This shouldn't be capitalized